src/System.Windows.Forms.Primitives/src/Windows/Win32/UI/Controls/LVITEMW.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Windows.Win32.UI.Controls;
+
+internal partial struct LVITEMW
+{
+    /// <summary>
+    /// Set the new text. The text length is limited by <see cref="cchTextMax"/>.
+    /// A value of <see cref="cchTextMax"/> will be updated to the length of <paramref name="text"/> + 1.
+    /// </summary>
+    /// <param name="text">The text to set.</param>
+    public unsafe void UpdateText(ReadOnlySpan<char> text)
+    {
+        if (cchTextMax <= text.Length)
+        {
+            text = text[..(cchTextMax - 1)];
+        }
+        else
+        {
+            cchTextMax = text.Length + 1;
+        }
+
+        // Create a span from the pszText and copy the input text to it
+        Span<char> targetSpan = pszText.AsSpan()[..cchTextMax];
```

Comment:
Okay so I'm not sure why the CI points to the failing test as being `LVITEMWTests.UpdateText_should_limit_input_text_to_cchTextMax_less_1_text_longer` , but I found that the real test that is failing/aborting is [this one](https://github.com/dotnet/winforms/blob/f59965e707d1a68bf381f59a721a97ff0c6fbcd2/src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewTests.cs#L4374). I think we will need to revert this back to what this originally was to resolve this. I am seeing that in the test `pszText` points to an empty string so this change results in `OutOfRangeException`.
```suggestion
        Span<char> targetSpan = new(pszText, cchTextMax);
```